guide/source/2.8-migration.md

```diff
@@ -0,0 +1,249 @@
+
+<h3 id="why-like-this">Why this is important?</h3>
+
+You may know that on Meteor we use a package called [Fibers](https://github.com/laverdet/node-fibers). This package is what makes possible to call async function, like `db.findOne()`, inside Meteor in a sync way (without having to wait for the promise to resolve).  
+
+But starting from Node 16, Fibers will stop working, so Meteor needs to move away from Fibers, otherwise, we'll be stuck on Node 14.
+
+If you want to know more about the plan, you can check this [discussion](https://github.com/meteor/meteor/discussions/11505).
+
+<h3 id="why-now">Why doing this now?</h3>
+
+This will be a considerable change for older Meteor applications, and some parts of the code of any Meteor app will have to be adjusted eventually. So it's important to start the migration process now.
+
+With this version, you'll be able to start preparing your app for the future by replacing your current MongoDB methods with the new async ones.
+
+<h3 id="what-is-new">What's new?</h3>
+
+Here are the newly added methods (you can see this description and the code [here](https://github.com/meteor/meteor/pull/12028)):
+
+**Added async methods on collections.**
+ - All async methods have an Async suffix to their names. These are: `createCappedCollectionAsync`, `createIndexAsync`, `dropCollectionAsync`, `dropIndexAsync`, `findOneAsync`, `insertAsync`, `removeAsync`, `updateAsync`, and `upsertAsync`.
+
+**Added async methods on cursors.**
+ - All async methods have an Async suffix to their names. These are: `countAsync`, `fetchAsync`, `forEachAsync`, and `mapAsync`.
+ - There's also `[Symbol.asyncIterator]`, so this code should work:
+    ```js
+    for await (const document of collection.find(query, options)) /* ... */
+    ```
+
+There is also a new Meteor method called `callAsync`. It should be used to call async methods.
+
+<h3 id="how-do-i-update">How can I start using these new features?</h3>
+
+We got a few examples for making it easy to use these new feature, you can look the code snippet bellow
+
+```js
+// SERVER
+
+// Before 2.8, we would use something like this
+export const removeByID = ({ id }) => {
+  SomeCollection.remove({ _id: id });
+};
+
+// Now we can also do like this
+export const removeByIDAsync = async ({ id }) => {
+  await SomeCollection.removeAsync({ _id: id });
+};
+
+Meteor.methods({
+   //...
+   removeByID,
+   removeByIDAsync,
+});
+
+// CLIENT
+
+const result = Meteor.call('removeByID', { id });
+
+// For the async, you call it like this:
+
+const result = await Meteor.callAsync('removeByIDAsync', { id });
+
+// or even like this:
+
+Meteor.callAsync('removeByIDAsync', { id }).then(result => {
+   console.log(result);
+});
+```
+
+More examples can be retrieved from [this commit](https://github.com/fredmaiaarantes/simpletasks/compare/main...mongodb-async-api).
```

Comment:
This link is comparing one branch to another, but this URL can be broken once the second branch no longer exists. I think we should remove this URL from here and if there are good examples there we could bring them into this guide.